tf_dependency/tf_configure.bzl

```diff
@@ -183,19 +185,22 @@ def _tf_pip_impl(repository_ctx):
     )
 
     tf_shared_library_dir = repository_ctx.os.environ[_TF_SHARED_LIBRARY_DIR]
-    tf_shared_library_path = "%s/libtensorflow_framework.so.2" % tf_shared_library_dir
+    tf_shared_library_name = repository_ctx.os.environ[_TF_SHARED_LIBRARY]
```

Comment:
Can we use `_TF_SHARED_LIBRARY_NAME` for consistency?

Reply:
Yeah makes sense. Thanks.